src/index.html

```diff
@@ -7,12 +8,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
```

Comment:
It seems like the `href` attribute for the preconnect links to Google Fonts is missing the URL. You should provide the full URL to establish the preconnect.